working-groups.md

```diff
@@ -399,6 +400,38 @@ TBD [community#629](https://github.com/tektoncd/community/issues/629)).
 | <img width="30px" src="https://github.com/afrittoli.png">  | Andrea Frittoli | IBM     | [afrittoli](https://github.com/afrittoli)   |
 | <img width="30px" src="https://github.com/xchapter7x.png"> | John Calabrese  | Google  | [xchapter7x](https://github.com/xchapter7x) |
 
+## Results
+
+This is the working group for [tektoncd/results](https://github.com/tektoncd/results).
+This should not be confused with the [TEP-0086 Larger Results](#tep-0086-larger-results) working group.
+
+| Artifact | Link                                                              |
+|----------|-------------------------------------------------------------------|
+| Forum    | [tekton-dev@](https://groups.google.com/forum/#!forum/tekton-dev) |
+| Meeting Notes              | TBD |
+| Slack Channel              | [#results](https://tektoncd.slack.com/messages/results) |
+
+### Europe + North America
+
+| Artifact | Link                                                              |
+|----------|-------------------------------------------------------------------|
+| Community Meeting VC       | TBD |
+| Community Meeting Calendar | Thursday every 4 weeks, 9:30a PST (alternates with Europe + Asia)<br>Link TBD|
+
+| &nbsp;                                                     | Facilitators    | Company | Profile                                       |
+|------------------------------------------------------------|-----------------|---------|-----------------------------------------------|
+| <img width="30px" src="https://github.com/adambkaplan.png">| Adam Kaplan     | Red Hat | [adambkaplan](https://github.com/adambkaplan) |
```

Comment:
@dibyom I imagine you'll want to be on the facilitator rotation for the Europe + NA meeting?

Reply:
Sure, I can help out.

Reply:
Maybe @alan-ghelardi will be interested as well?